mne/viz/tests/test_topomap.py

```diff
@@ -568,6 +568,24 @@ def patch():
     assert_array_equal(evoked_grad.info["bads"], orig_bads)
 
 
+def test_plot_psds_topomap_colorbar():
+    """Test plot_psds_topomap colorbar option."""
+    raw = read_raw_fif(raw_fname)
+    picks = pick_types(raw.info, meg="grad")
+    info = pick_info(raw.info, picks)
```

Comment:
[nitpick/FYI]: there is `mne.io.read_info(raw_fname)` for cases where you don't need the actual raw data. Here you need `picks` too, so you'd also need `mne.channel_indices_by_type()` so in the end it's not any shorter:

```suggestion
    info = read_info(raw_fname)
    picks = channel_indices_by_type(info)["grad"]
    info = pick_info(info, picks)
```

and you might need to add `from mne.io import read_info` and `from mne import channel_indices_by_type` to the top of the file.

Reply:
Good point, will keep this in mind!